src/main/java/org/apache/fineract/config/PaymentModeConfiguration.java

```diff
@@ -0,0 +1,27 @@
+package org.apache.fineract.config;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.stereotype.Component;
+import java.util.ArrayList;
+import java.util.List;
+
+@Getter
+@Setter
+@NoArgsConstructor
```

Comment:
we can skip @NoArgsConstructor if there is no explicit constructor,   NoArgsConstructor is a java default behaviour

Reply:
it gives me error. Saying no empty constructor found.

Reply:
Need to check this. It should add it by default